user_manual/files/encrypting_files.rst

```diff
@@ -105,3 +105,39 @@ login password using that back-end configuration. In this case, you can set
 your encryption password to your new login password by providing your old and
 new login password. The Encryption app works only if your login password and
 your encryption password are identical.
+
+How to Restore Files From Backup When Encryption Is Enabled
+-----------------------------------------------------------
+
+If you need to restore files from backup, which were backed up when encryption
+was enabled, here’s how to do it.
+
+.. NOTE:: 
+   This is effective from at least version v8.2.7 of ownCloud onwards. Also,
+   this is **not officially supported**. ownCloud officially supports either
+   restoring the full backup or restoring nothing — not restoring individual
+   parts of it.
+
+1. Restore the file from backup
+2. Restore the file's encryption keys from backup
+3. Run ``occ files:scan``; this makes the scanner find it. Note that, in the DB
```

Comment:
@settermjd I think this should be in the admin manual. Users do not have access to the CLI and cannot restore backups. Can you move it ? (I can make a ticket if you like)

Reply:
I moved it in (73a16138). Documenting it here, as I had to search for the information, and don't want to do that again, if I can avoid it.